apps/wfo-ui-surf/components/WfoAvatar/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
+import { css } from '@emotion/react';
+
+export const getStyles = (theme: EuiThemeComputed) => {
+    const stepStateIcon = {
+        height: '42px',
+        width: '42px',
+        alignItems: 'center',
+        display: 'flex',
+        justifyContent: 'center',
+        borderRadius: '24px',
+        border: `2px solid ${theme.colors.emptyShade}`,
+    };
+
+    const openIconStyle = css({
+        ...stepStateIcon,
+        backgroundColor: '#D9EDE1',
```

Comment:
Are these colors not part of the theme? This is surf specific but I would still expect everything to follow the general design

Reply:
These colors are also used in the WfoWorkflowSteps and they are also not part of the theme.

Reply:
would be good to use the theme, maybe also fix this in other parts of the code?